drizzle-orm/src/built-in-function.ts

```diff
@@ -0,0 +1,63 @@
+import { entityKind } from './entity.ts';
+import type { Dialect } from './column-builder.ts';
+import type { SQLWrapper, SQL, DriverValueDecoder, GetDecoderResult } from './sql/sql.ts';
+
+/** @internal */
+export const BuiltInFunctionSQL = Symbol.for('drizzle:BuiltInFunctionSQL');
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export interface BuiltInFunction<T = unknown> extends SQLWrapper {
+  // SQLWrapper runtime implementation is defined in 'sql/sql.ts'
+}
+export abstract class BuiltInFunction<T = unknown> implements SQLWrapper {
```

Comment:
Hey Mario, why do you think this new class is necessary?
It seems like it's only used to hold the `sql` value and the `as` and `mapWith` methods are the same as the original `sql` operator?

Reply:
This class has been removed and now the aggregate functions are just functions that wrap the `sql` operator.